### DISCUSSION TITLE
Debate on converting images to Base64

### DISCUSSION THREAD

**Discussion by kcramakrishna** [Digital-Assistant/Digital_Assistant_Client#389] [General]
Since we have been using AntD, we converted the images to base64 and folding them into the single js file that we are serving.
This has pros and cons. The biggest con is that we cannot cache images. We need to dig deeper and check on why AntD does this in the first place and if we should override the default behaviour. 

Depending on the outcome of the research, we should take a decision on whether to continue to base64 or move from base64 to png/jpg.

https://stackoverflow.com/questions/11736159/advantages-and-disadvantages-of-using-base64-encoded-images
https://stackoverflow.com/questions/1574961/how-much-faster-is-it-to-use-inline-base64-images-for-a-web-site-than-just-linki
https://bunny.net/blog/why-optimizing-your-images-with-base64-is-almost-always-a-bad-idea/

google of "why AntD uses base64" did not give me any answers.

**Comment by yureshwar** [comment]
https://www.keycdn.com/support/image-base64-encoding

According to this using it as seperate calls for larger images it better. But if we want to decrease the number of calls we should use base64 based images
